### ISSUE TITLE
Marlin 1.1.1 Stable releases ==> FW K8400 V1.6  ??

### ISSUE THREAD

**Issue by LuckyLuzz** [birkett/Vertex-K8400-Firmware#2]
Hallo Birkett,
I am using your Branch of Marlin and I was wondering if you are going to upgrade to V1.6 with the new stable Marlin 1.1.1?

**Comment by birkett** [comment]
Yes - I want to get a 1.1x branch going soon. Upstream Marlin kicked their heels for over a year in RC phase, so I missed the actual release.

**Comment by pmbdk** [comment]
This would really be awesome!

**Comment by pmbdk** [comment]
Btw, it seems that Marlin 1.1 already contains example configuration for the Vertex/K8400:

https://github.com/MarlinFirmware/Marlin/tree/1.1.x/Marlin/example_configurations/K8400

**Comment by birkett** [comment]
Indeed, I up streamed the configuration some time ago. Vanilla Marlin works well on the K8400, but lacks the LED controls and custom menu text.

**Comment by pmbdk** [comment]
Nice! I can live without led control and custom menu text :-)

**Comment by EcoApi** [comment]
Hello birkett.
have you an rc of marlin 1.1.x ? Auto bed leveling was updeted in this version of Marlin.

**Comment by creisey** [comment]
i have installed the latest marlin but it lacks of filament change from the menu. its only available during print... how do you manage that?

**Comment by hapklaar** [comment]
> i have installed the latest marlin but it lacks of filament change from the menu. its only available during print... how do you manage that?

Check out https://www.youtube.com/watch?v=CDz2IxxINBE&feature=youtu.be on how to enable this in fw

**Comment by Knoose2** [comment]
I am having a compilation error in arduino for marlin 1.1.9, it says missing binary operator before token "("

Tried compiling with 1.6.9 and 1.6.11 and latest version 1.6.23.

Edit: Solved, arduino install was broken on pc, reinstall did not help but had a laptop lying aroudn with clean install. Compiled and uploaded correctly

**Comment by Knoose2** [comment]
Also worth mentioning is that I needed to change some settings so it could recognise my heated bed (select correct thermistor) and I believe it was set to a filament diameter of 3mm instead of 1.75.

Most of you porbably know this, but some might not :)
